HW_3/templates/base.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    {% block title %}
+    <title>
+        Index
+    </title>
+    {% endblock %}
+</head>
+{% block body %}
+<body>
+    <h1>
+        Main page.
+    </h1>
+</body>
+{% endblock %}
```

Comment:
Все, что между началом и концом блока заменится дочерним шаблоном. Поэтому блок нужно помещать внутри тега `body`, потому что дочернем шаблоне его нет.
```suggestion
<body>
{% block body %}
    <h1>
        Main page.
    </h1>
{% endblock %}
</body>
```